app/services/dashboard/moab_on_storage_service.rb

```diff
@@ -143,7 +144,7 @@ def num_moab_record_not_ok
 
     def num_preserved_objects
       # used multiple times, so memoize to avoid db queries
-      @num_preserved_objects ||= PreservedObject.count
+      @num_preserved_objects ||= PreservedObject.all.annotate(sa).count
```

Comment:
is `.all` useful?

Reply:
Yes, it is required for `.annotate`. This is clarified in README update.